values.md

```diff
@@ -1,22 +1,25 @@
 # Rockefeller Archive Center Digital Program Values
 
-The values of the Rockefeller Archive Center's Digital Program inform how we work together as a team, as well as how we interact with RAC staff, researchers, donors, colleagues and other external partners. These values support and inform our strategic goals and provide a standard by which we measure our daily practice. We respect and contribute to a vibrant culture based on these values, and raise concerns clearly when these values are not being followed. These values are reviewed on an annual basis and revised as necessary. 
+The values of the Rockefeller Archive Center's Digital Program inform how we work together as a team, as well as how we interact with RAC staff, researchers, donors, colleagues and other external partners. These values support and inform our strategic goals and provide a standard by which we measure our daily practice. We respect and contribute to a vibrant culture based on these values, and raise concerns clearly when these values are not being followed. These values are reviewed on an annual basis and revised as necessary.
 
 ## Engage and empower users.
 
-We serve multiple constituencies and always work for the good of others. Engage with users to understand their needs, prioritize solutions, and produce outcomes that provide high-quality and useful services for researchers, donors and RAC staff. Provide the resources, training, and support necessary for users to feel empowered to use and troubleshoot our systems.
+We serve multiple constituencies and always work for the good of others. Engage with users to understand their needs, prioritize solutions, and produce outcomes that provide high-quality and useful services for researchers, donors and RAC staff. Provide the care, resources, training, and support necessary for users to feel empowered to use and troubleshoot our systems.
```

Comment:
Yeah, in this case support is not technical, so it really is care. I'd be behind this change.